webgme/webgme-blob.d.ts

```diff
@@ -0,0 +1,15 @@
+
+
+declare namespace Blob {
+
+    export type ObjectBlob = string;
+
+    export interface BlobMetadata {
+        name: string;
+        size: number;
+        mime: string;
+        context: Core.DataObject;
+        contentType: string;
+    }
+    export interface Util { }
```

Comment:
An empty interface is equivalent to the `{}` type. Implementing it adds no information.